src/django/api/management/commands/delete_contributor.py

```diff
@@ -0,0 +1,42 @@
+import sys
+
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from allauth.account.models import EmailAddress
+
+from api.models import User, Contributor
+
+
+class Command(BaseCommand):
```

Comment:
I would like to consider giving this the more detailed name `delete_user_and_contributor` as a way of reinforcing that we are deleting multiple things.

Reply:
Sounds good. I renamed it in 
26f33df